src/App.jsx

```diff
@@ -19,8 +19,6 @@ export const goods = [
 export const App = () => {
   const [good, setGood] = useState('Jam');
```

Comment:
It would be nice to add a constant variable like DEFAULT_GOOD and store 'Jam' here